core/math/math_funcs.h

```diff
@@ -253,6 +253,35 @@ class Math {
 						(-p_pre + 3.0f * p_from - 3.0f * p_to + p_post) * (p_weight * p_weight * p_weight));
 	}
 
+	static _ALWAYS_INLINE_ double cubic_interpolate_angle(double p_from, double p_to, double p_pre, double p_post, double p_weight) {
```

Comment:
I implemented this for consistency. However, Animation uses `cubic_interpolate_angle_in_time()`, so I am not sure if this is necessary.